src/app/containers/FrontPageSection/index.test.jsx

```diff
@@ -153,17 +153,17 @@ describe('FrontPageSection Container', () => {
 
     shouldShallowMatchSnapshot(
       'should render correctly for canonical',
-      <FrontPageSection group={group} />,
+      <FrontPageSection group={group} sectionNumber={0} />,
```

Comment:
Sorry, I know you changed these to make them test when `sectionNumber === 0`, could we also have one extra snapshot test where `sectionNumber === 1`? (or your favourite non-zero number)

Reply:
One of the snapshots now has `sectionNumber === 1`